### PR TITLE
Seed of Power

### PR DESCRIPTION
**Pull request by TectonicRifts**
Quest Seed of Power from 2013-7 Feelings of Dread.

Set correct spell projectile for 05969 Galvanic Strike.

Set correct destination for spells 06148 Begone and Be Afraid and 06149 Rynthid Vision.

### CODE REVIEW

File: Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+     , (51599,   6,         -1) /* ItemsCapacity */
+     , (51599,   7,         -1) /* ContainersCapacity */
+     , (51599,  16,          1) /* ItemUseable - No */
+     , (51599,  25,        200) /* Level */
+     , (51599,  93,       3084) /* PhysicsState - Ethereal, ReportCollisions, Gravity, LightingOn */
+     , (51599, 133,          4) /* ShowableOnRadar - ShowAlways */
+     , (51599, 146,    1100000) /* XpOverride */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (51599,   1, True ) /* Stuck */
```
Comment (DridsGhost) [inline Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql:18]: Should these be attackable false?

File: Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+     , (51599, 125,       1) /* ResistHealthDrain */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (51599,   1, 'Pillar of Lightning') /* Name */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (51599,   1,   33560373) /* Setup */
+     , (51599,   2,  150995290) /* MotionTable */
+     , (51599,   3,  536871001) /* SoundTable */
+     , (51599,   4,  805306379) /* CombatTable */
```
Comment (DridsGhost) [inline Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql:64]: Getting combatmaneuver errors on tehse guys

**Review by DridsGhost — COMMENTED** [review]
I can run right past the pillars of lightning, they dont cast on me unless I stop and hang around. Seems odd

File: Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql (hunk 1 above)
Comment (TectonicRifts) [reply Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql:18]: They show as yellow dots in the video so I assume can be attacked.

File: Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql (hunk 2 above)
Comment (TectonicRifts) [reply Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Elemental/51599 Pillar of Lightning.sql:65]: Can you test these again? I stopped getting the error but can't explain why. The casting delay seems to have been taken care of by making them AI immobile.

File: Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Unsorted/51554 Door.sql [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+/* @teleloc 0x587101E1 [154.750000 -50.000000 -0.068000] -0.707107 0.000000 0.000000 -0.707107 */;
+
+INSERT INTO `weenie_properties_attribute` (`object_Id`, `type`, `init_Level`, `level_From_C_P`, `c_P_Spent`)
+VALUES (51554,   1,  50, 0, 0) /* Strength */
+     , (51554,   2,  50, 0, 0) /* Endurance */
+     , (51554,   3,  50, 0, 0) /* Quickness */
+     , (51554,   4,  50, 0, 0) /* Coordination */
+     , (51554,   5,  50, 0, 0) /* Focus */
+     , (51554,   6,  30, 0, 0) /* Self */;
+
```
[the remark this answers is not in the document]
Comment (TectonicRifts) [reply Database/Patches/2013-07-FeelingsOfDread/9 WeenieDefaults/Creature/Unsorted/51554 Door.sql]: I was missing a bunch of stuff on these. I made them closely match the door from Gaerlan, with skills, body table and other properties.

**Review by DridsGhost — APPROVED** [review]

**Pull request merged**